DataProcessing/datax-host/src/main/scala/datax/fs/HadoopClient.scala

```diff
@@ -361,6 +361,9 @@ object HadoopClient {
         case e: TimeoutException =>
           remainingAttempts = 0
           throw e
+        case e: IOException =>
```

Comment:
Lets also add, AppInsightLogger.trackException(e, Map(
          "errorLocation" -> "writeWithTimeoutAndRetries",
          "errorMessage" -> "Error in writing file",
          "failedHdfsPath" -> hdfsPath
        ), null)

Reply:
added this in both IOException and TimeoutException